Specs/Number/French/NumberModel.json

```diff
@@ -1661,7 +1661,7 @@
   },
   {
     "Input": "Je veux un crédit de €1000 sur 3 ans",
-    "NotSupported": "python",
+    "NotSupported": "java, python",
```

Comment:
Why?

Reply:
In my local test, this sentence is failed in Java and python. I tried to figure out what caused this error, but I got nothing...